apps/src/weblab/WebLabView.jsx

```diff
@@ -152,6 +163,24 @@ class WebLabView extends React.Component {
               style={iframeStyles}
             />
             {!this.props.isProjectLevel && <CompletionButton />}
+            <BaseDialog
+              isOpen={this.props.shouldShowError}
+              handleClose={this.closeErrorDialog}
+            >
+              <h1>{weblabMsg.uploadError()}</h1>
+              <p>{weblabMsg.errorSavingProject()}</p>
+              <div style={{position: 'relative'}}>
+                <Button
+                  onClick={() => window.location.reload()}
```

Comment:
this can be simplified to:

```suggestion
                  onClick={window.location.reload}
```

Reply:
For window.location.reload, we still need the arrow function. I believe it's required in order to handle binding. If anybody has a handy article explaining this, that would be awesome. Everything I've found so far just says you need to do it this way without a clear explanation of why.

Reply:
ohhhh, that makes sense. i can't find anything explaining this either, but in react you have to manually bind ES5 functions, but ES6 functions don't need arrow functions, so it could be something along those lines